.github/workflows/community.yml

```diff
@@ -39,7 +39,8 @@ jobs:
           python-version-file: '.python-version'
           cache: "pip"
 
-      - run: pip install -r requirements.txt
+      - run: pip install --user pipenv
+      - run: pipenv install
```

Comment:
I think it make sense for the CI to use `--deploy` option, see https://pipenv-fork.readthedocs.io/en/latest/basics.html#pipenv-install